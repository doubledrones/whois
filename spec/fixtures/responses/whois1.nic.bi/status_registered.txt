Domain Name: google.bi
Domain ID: 2633NIC-BI
WHOIS Server: whois1.nic.bi
Referral URL: 
Updated Date: 2013-05-20T14:10:41.854Z
Creation Date: 2002-09-29T22:00:00.000Z
Registry Expiry Date: 2013-09-29T22:00:00.000Z
Sponsoring Registrar: Afriregister
Sponsoring Registrar IANA ID: 
Domain Status: ok

Registrant ID: 82159NIC-BI
Registrant Name: DNS Admin
Registrant Organization: Google Inc.
Registrant Street: 1600 Amphitheatre Parkway
Registrant City: Mountain View
Registrant State/Province: CA
Registrant Postal Code: 94043
Registrant Country: US
Registrant Phone: +1.6506234000
Registrant Phone Ext: 
Registrant Email: dns-admin@google.com

Admin ID: 82159NIC-BI
Admin Name: DNS Admin
Admin Organization: Google Inc.
Admin Street: 1600 Amphitheatre Parkway
Admin City: Mountain View
Admin State/Province: CA
Admin Postal Code: 94043
Admin Country: US
Admin Phone: +1.6506234000
Admin Phone Ext: 
Admin Email: dns-admin@google.com

Billing ID: 82160NIC-BI
Billing Name: MM Domain Administrator
Billing Organization: MarkMonitor Inc
Billing Street: 391 N Ancestor Place
Billing City: Boise
Billing State/Province: Idaho
Billing Postal Code: 83704
Billing Country: US
Billing Phone: +1.2083895740
Billing Phone Ext: 
Billing Email: ccopsbilling@markmonitor.com

Tech ID: 82159NIC-BI
Tech Name: DNS Admin
Tech Organization: Google Inc.
Tech Street: 1600 Amphitheatre Parkway
Tech City: Mountain View
Tech State/Province: CA
Tech Postal Code: 94043
Tech Country: US
Tech Phone: +1.6506234000
Tech Phone Ext: 
Tech Email: dns-admin@google.com

Name Server: ns1.google.com
Name Server: ns2.google.com
Name Server: ns3.google.com
Name Server: ns4.google.com

DNSSEC: unsigned


Additional Section


Sponsoring Registrar Country: BI
Sponsoring Registrar Admin Contact: 
Sponsoring Registrar Admin Email: 


TERMS OF USE: You are not authorized to access or query our Whois database through the use of electronic processes that are high-volume and automated.  Whois database is provided by NIC.BI as a service to the internet community on behalf of Burundi cctld.)

The data is for information purposes only. NIC.BI does not guarantee its accuracy. By submitting a Whois query, you agree to abide by the following terms of use: You agree that you may use this Data only for lawful purposes and that under no circumstances will you use this Data to: (1) allow, enable, or otherwise support the transmission of mass unsolicited, commercial advertising or solicitations via e-mail, telephone, or facsimile; or (2) enable high volume, automated, electronic processes

The compilation, repackaging, dissemination or other use of this Data is expressly prohibited.


>>> Last update of WHOIS database: 2013-06-30T15:00:06.689Z <<<
