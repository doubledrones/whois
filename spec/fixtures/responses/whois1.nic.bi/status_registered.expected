#domain
  should: %s == "google.bi"

#domain_id
  should: %s == "2633NIC-BI"


#status
  should: %s == :registered

#available?
  should: %s == false

#registered?
  should: %s == true


#created_on
  should: %s CLASS(time)
  should: %s == Time.parse("2002-09-29 22:00:00 UTC")

#updated_on
  should: %s CLASS(time)
  should: %s == Time.parse("2013-05-20 14:10:41 UTC")

#expires_on
  should: %s CLASS(time)
  should: %s == Time.parse("2013-09-29 22:00:00 UTC")


#registrar
  should: %s CLASS(registrar)
  should: %s.id           == nil
  should: %s.name         == "Afriregister"
  should: %s.url          == nil


#nameservers
  should: %s CLASS(array)
  should: %s SIZE(4)
  should: %s[0] CLASS(nameserver)
  should: %s[0].name == "ns1.google.com"
  should: %s[1] CLASS(nameserver)
  should: %s[1].name == "ns2.google.com"
  should: %s[2] CLASS(nameserver)
  should: %s[2].name == "ns3.google.com"
  should: %s[3] CLASS(nameserver)
  should: %s[3].name == "ns4.google.com"
